src/contracts/core/AVSDirectory.sol

```diff
@@ -232,15 +289,13 @@ contract AVSDirectory is
             "AVSDirectory.registerOperatorToAVS: operator signature expired"
         );
 
-        MemberInfo storage member = memberInfo[msg.sender][operator];
+        // Assert that the AVS is not an operator set AVS.
+        require(!isOperatorSetAVS[msg.sender], "AVSDirectory.registerOperatorToAVS: AVS is an operator set AVS");
```

Comment:
we should put this in the docs. registration is bricked once migration starts

Reply:
Added here: https://github.com/Layr-Labs/eigenlayer-contracts/pull/631/commits/c3b723b42895361f2e1b6aaec2e135944161ca2d